public/components/custom_panels/panel_modules/visualization_container/visualization_container.tsx

```diff
@@ -180,7 +187,7 @@ export const VisualizationContainer = ({
       disabled={editMode}
       onClick={() => {
         closeActionsMenu();
-        if (visualizationMetaData?.subType === PROMQL_METRIC_SUBTYPE) {
+        if (visualizationMetaData?.metricType === PROMQL_METRIC_SUBTYPE) {
```

Comment:
this would break backward compatibility and change behavior for old schema with `subType` not `metricType`?

Reply:
No it wouldn't. Please refer this comment - https://github.com/opensearch-project/dashboards-observability/pull/1314/files#r1442453061